SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -26,6 +58,10 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
 	TSubclassOf<USpatialMultiWorkerSettings> MultiWorkerSettingsClass;
 
+	/** Defines how Unreal Editor will run the Tests in this map, without changing current Settings. */
```

Comment:
We could wrap this in `WITH_EDITORONLY_DATA`

Reply:
When I put it as advanced, since it is alone in that category it will just make that setting appear lower with a expansion button that isn't clickable. Are we sure we want it like this?
![image](https://user-images.githubusercontent.com/5429121/89406877-bb9af080-d715-11ea-8801-a8907b9144e6.png)

Reply:
So to be clear without advanced it appears closer to the top (after Lightmass):
![image](https://user-images.githubusercontent.com/5429121/89407188-3c59ec80-d716-11ea-94b1-21e95a6d8bbf.png)

and without that unclickable expand button, and with advanced it looks like that and the section will appear further down after Authority section:
![image](https://user-images.githubusercontent.com/5429121/89407065-0b79b780-d716-11ea-8354-147b004a77c3.png)

Reply:
I prefer without advanced

Reply:
> When I put it as advanced, since it is alone in that category it will just make that setting appear lower with a expansion button that isn't clickable. Are we sure we want it like this?
I think this will just be because you have your defaults set to view advanced. If you put it to view defaults, it should hide advanced settings.

Reply:
Like we discussed offline, that was even with the view defaults so for now assuming we're leaving AdvancedDisplay off.